types/vscode-proposed/index.d.ts

```diff
@@ -173,18 +173,46 @@ export interface NotebookDocument {
     readonly uri: Uri;
     readonly version: number;
 
+    /** @deprecated Use `uri` instead */
```

Comment:
Why do we seem to have three proposed API files? I only updated types/vscode.proposed.d.ts in my last API change.

Reply:
I'll remove the others in the separate PR.